src/plugins/intel_gpu/src/graph/impls/ocl_v2/moe_expert_down_cm.cm

```diff
@@ -0,0 +1,242 @@
+// Copyright (C) 2025 Intel Corporation
```

Comment:
Please move cm kernels to src/plugins/intel_gpu/src/graph/impls/cm

Reply:
Done.

Reply:
updated!